app/views/ViewDates.scala

```diff
@@ -16,18 +16,23 @@
 
 package views
 
-import java.time.{ZoneId, ZoneOffset}
+import java.time.ZoneId
 import java.time.format.DateTimeFormatter
 import java.time.temporal.TemporalAccessor
 
 object ViewDates {
 
-  val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
-  val submissionDateTimeFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM uuu 'at' HH:mm").withZone(ZoneId.of("Europe/London"))
-  val dateFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM uuu").withZone(ZoneId.of("Europe/London"))
+  private val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
+  private val submissionDateTimeFormatter: DateTimeFormatter =
+    DateTimeFormatter.ofPattern("d MMMM uuu 'at' h:mma").withZone(ZoneId.of("Europe/London"))
```

Comment:
`mma`? what `a` means? Is it for `am` and `pm`?

Reply:
Yes.  But it produces "AM" and "PM" unfortunately - no option for lower-case - hence the string replacement